### ISSUE TITLE
<eventDescription>

### ISSUE THREAD

**Issue by SJagodzinski** [SAA-SDT/eac-cpf-schema#96]
## Maintenance Event Description
- add optional attributes
`@audience`
`@scriptOfElement`
- add optional child element< `<reference>`, `<span>`
- keep element definition

## Creator of issue
1. Silke Jagodzinski
2. TS-EAS: EAC-CPF subgroup
3. silkejagodzinski@gmail.com

## Related issues / documents
[new element `<evidence>` #73](https://github.com/SAA-SDT/eac-cpf-schema/issues/73
[Topic_assertion_control.pdf](https://github.com/SAA-SDT/TS-EAS-subteam-notes/blob/master/eaccpf-subteam/working-documents/topics/assertion/Topic_assertion_control.pdf)
[`<maintenanceEvent>` #91](https://github.com/SAA-SDT/eac-cpf-schema/issues/91)

## EAD3 Reconciliation
**Additional EAD 3 attributes**
Attributes:
`@altrender` - Optional
`@audience` - Optional (values limited to: external, internal)
`@encodinganalog` - Optional
`@localtype` - Optional
`@script` - Optional

## Context
The description of a maintenance event in the life of the EAC-CPF instance.

**May contain**: [text]
**May occur within**: `<maintenanceEvent>`
**Attributes:** `@xml:id`, `@xml:lang` - all optional
**Availability:** Optional, Repeatable

## Solution documentation
Rephrasing _Summary_, _Description and Usage_ and _Attribute usage_ needed?

**May contain**: [text], `<reference>` (0..n), `<span>` (0..n)
**May occur within**: `<maintenanceEvent>`
**Attributes:**
`@audience` - optional (values limited to: external, internal)
`@id` - optional 
`@languageOfElement` - optional 
`@scriptOfElement` - optional 

**Availability:** Optional, repeatable

- New or other example needed
- Topic for Best Practise Guide

## Example encoding
```
<control>
 <recordId>records identifier</recordId>
 <maintenanceAgency>  [...] </maintenanceAgency> 
 <maintenanceHistory>
  <maintenanceEvent>
   <eventDateTime>01.07.2020</eventDateTime>
   <agent>agent name</agent>
   <eventDescription audience="external" id="eventdescription1" languageOfElement="de" scriptOfElement="lat">description of the maintenance event</eventDescription>
  </maintenanceEvent>
 </maintenanceHistory>
</control>
```

**Comment by SJagodzinski** [comment]
EAC-CPF first [November 2020 meeting](https://github.com/SAA-SDT/TS-EAS-subteam-notes/blob/master/eaccpf-subteam/meeting-minutes/2020/20201106_EAC-CPF_minutes.pdf):

> [...] agreed to have no attribute to point from a maintenance event to a source

**Comment by ailie-s** [comment]
Tag Library Text:

**Summary:** An optional child element of `<maintenanceEvent>` that provides the description of a maintenance event in the life of the EAC-CPF instance.
**May contain:** [text], reference (0..n), span (0..n)
**May occur within:** maintenanceEvent
**Attributes:**
`@audience `- optional (values limited to: external, internal)
`@id` - optional
`@languageOfElement` - optional
`@scriptOfElement `- optional
`@target` - optional
**Description and Usage:**
A child element of `<maintenanceEvent>` used for describing the maintenance event. The element allows a full description of the maintenance event to be given alongside information about the maintenance agent, type and time and date of the maintenance event.
**Availability:** Optional, repeatable

**Comment by kerstarno** [comment]
Tested as part of Schema Team's schema testing:

- `<eventDescription>` is an optional sub-element of `<maintenanceEvent>`
- `<eventDescription>` can be repeated
- `<eventDescription>` can contain text
- `<eventDescription>` does currently not allow for sub-elements `<span>` and `<reference>` (as per the decision during the EAC Team meeting on 8 January)
  - I've updated the control.rng accordingly
  - I've then re-tested and can now confirm that `<eventDescription>` does allow for `<span>` and `<reference>` as repeatable sub-elements
- `<eventDescription>` allows for the optional attributes `@audience`, `@id`, `@target`, `@languageOfElement`, and `@scriptOfElement`
- `<eventDescription>` also allows for optional attributes from other namespaces (tested with XLink)

The above applies to the RNG and the XSD schemas.

The element is ready.

@ailie-s - the TL draft above does not yet account for `<span>` and `<reference>` as sub-elements of `<eventDescription>`.
